### PR TITLE
Javascript

### PR DESCRIPTION
**Pull request by abolayo**
i linked my landing page to a product page
i populate the product page using javascript

### CODE REVIEW

File: submissions/medinatapanpa/javascript/dufuna.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
```
Comment (taiwo-ladipo) [inline submissions/medinatapanpa/javascript/dufuna.html]: Can you rename this file to convey what it does e.g registration?

File: submissions/medinatapanpa/javascript/js/nigerian.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+let nigerian = [
```
Comment (taiwo-ladipo) [inline submissions/medinatapanpa/javascript/js/nigerian.js]: Please use a descriptive variable name

**Review by taiwo-ladipo — CHANGES_REQUESTED** [review]
A lot of files from previous tasks were duplicated in this pull request. Please focus solely on the javascript task i.e loading products dynamically. Also, ensure the products are styled like an e-commerce site's products

**Comment by taiwo-ladipo** [comment]
It is not a good idea to delete files from previous task(s). Please revert

**Comment by abolayo** [comment]
please how do I revert the deleted files? but the files are not supposed to be part of this PR.
 how do I solve the conflict issue?

**Comment by tegaphilip** [comment]
> please how do I revert the deleted files? but the files are not supposed to be part of this PR.
> how do I solve the conflict issue?

@abolayo you need to do a revert to the last commit before you mistakenly deleted the files. or do a reset back to a previous commit. See https://code.likeagirl.io/how-to-undo-the-last-commit-393e7db2840b

if you need further help, ping me on slack, preferably from 9pm upwards or during the weekends.

**Pull request closed without merging**